.github/workflows/on_pr_master.yml

```
@@ -1,14 +1,36 @@
 name: CI/CD
-env:
-  GITHUB_TOKEN: ${{ secrets.PAT_GITHUB_TOKEN }}
 on:
   pull_request:
     branches: [ master ]
     types: [opened, synchronize, reopened, edited, closed]
```

Adding labeled would fix triggering github actions automatically when a PR is labeled after it is created.
```suggestion
    types: [opened, synchronize, reopened, edited, closed, labeled]
```